WcaOnRails/app/views/static_pages/organizations.html.erb

```diff
@@ -6,36 +6,24 @@
   <p><%= t('regional_organizations.content') %></p>
 
   <% organizations_info = [
-       {
-         country: "Africa",
-         name: "African Cubing Association",
-         url: "http://www.africancubing.org",
-         logo: "aca.jpg",
-       },
        {
          country: "Australia",
          name: "Speedcubing Australia",
          url: "http://www.speedcubing.org.au",
```

Comment:
This should be https://speedcubing.org.au/

Reply:
Fixed!